provision/core/phpcs/provision.sh

```diff
@@ -16,8 +16,9 @@ function php_codesniff_setup() {
   vvv_info " * Provisioning PHP_CodeSniffer (phpcs), see https://github.com/squizlabs/PHP_CodeSniffer"
 
   noroot mkdir -p /srv/www/phpcs
+  noroot cp -f "/srv/config/phpcs/composer.json" "/srv/www/phpcs/composer.json"
   cd /srv/www/phpcs
-  COMPOSER_BIN_DIR="bin" noroot composer require --update-with-all-dependencies "dealerdirect/phpcodesniffer-composer-installer" "wp-coding-standards/wpcs" "automattic/vipwpcs" "phpcompatibility/php-compatibility" "phpcompatibility/phpcompatibility-paragonie" "phpcompatibility/phpcompatibility-wp" --no-ansi --no-progress
+  COMPOSER_BIN_DIR="bin" noroot composer install --no-ansi --no-progress
 
   vvv_info " * Symlinking phpcs and phcbf into /usr/local/bin"
```

Comment:
should probably change this now we stopped symlinking

Reply:
Fixed